auslib/admin/views/permissions.py

```diff
@@ -180,6 +180,14 @@ def get(self, username):
         return jsonify({"roles": roles})
 
 
+class AllRolesView(AdminView):
+    """/users/roles"""
+
+    def get(self):
+        roles_list = dbo.permissions.getAllRoles()
+        return jsonify({"roles_list": roles_list})
```

Comment:
Nit: I think just plain "roles" would be a better name here - the type of the value is obvious to consumers.